private-templates-service/adaptivecards-templating-service/src/models/errorModels.ts

```diff
@@ -18,10 +18,12 @@ export class TemplateError {
  */
 export enum ApiError {
   InvalidTemplate = "InvalidTemplate",
+  InvalidQueryParam = "InvalidQueryParam",
   TemplateNotFound = "TemplateNotFound",
   InvalidAuthenticationToken = "InvalidAuthenticationToken",
   UserNotFound = "UserNotFound",
-  DeleteUserInfoFailed = "Failed to delete all user info."
+  DeleteUserInfoFailed = "Failed to delete all user info.",
```

Comment:
These are a different format to the ones above

Reply:
Oh I see, you've added them in the messages below - delete these?

Reply:
Thanks for the catch, I've updated the api errors to the right format